src/sentry/static/sentry/app/views/performance/landing/content.tsx

```diff
@@ -0,0 +1,219 @@
+            if (hasFeature) {
+              const additionalSummaryConditions = this.getSummaryConditions(
+                getAdditionalTableQuery(location)
+              );
+              const frontendEventView = generateFrontendPerformanceEventView(
+                this.props.organization,
+                this.props.location
+              );
+
+              const frontendTableEventView = frontendEventView.clone();
+              frontendTableEventView.query = `${summaryConditions} ${additionalSummaryConditions}`;
+              return (
+                <React.Fragment>
+                  <SearchContainer>
+                    <StyledSearchBar
+                      organization={organization}
+                      projectIds={frontendEventView.project}
+                      query={filterString}
+                      fields={generateAggregateFields(
+                        organization,
+                        [...frontendEventView.fields, {field: 'tps()'}],
+                        ['epm()', 'eps()']
+                      )}
+                      onSearch={handleSearch}
+                    />
+                    <ProjectTypeDropdown>
+                      <DropdownControl
+                        buttonProps={{prefix: t('Display')}}
+                        label={currentLandingDisplay.label}
+                      >
+                        {LANDING_DISPLAYS.map(({label, field}) => (
+                          <DropdownItem
+                            key={field}
+                            onSelect={this.handleLandingDisplayChange}
+                            eventKey={field}
+                            data-test-id={field}
+                            isActive={field === currentLandingDisplay.field}
+                          >
+                            {label}
+                          </DropdownItem>
+                        ))}
+                      </DropdownControl>
+                    </ProjectTypeDropdown>
+                  </SearchContainer>
+                  <VitalsCards
+                    eventView={frontendEventView}
+                    organization={organization}
+                    location={location}
+                    isAlwaysShown
+                  />
+                  {currentLandingDisplay.field === LandingDisplayField.FRONTEND && (
+                    <FrontendDisplay
+                      eventView={frontendEventView}
+                      organization={organization}
+                      location={location}
+                      onFrontendDisplayFilter={this.handleTableQueryUpdate}
+                    />
+                  )}
+                  <Table
+                    eventView={frontendTableEventView}
+                    projects={projects}
+                    organization={organization}
+                    location={location}
+                    setError={setError}
+                    summaryConditions={`${summaryConditions} ${additionalSummaryConditions}`}
```

Comment:
Couldn't this be `frontendTableEventView.query`?

Reply:
yeah it can, I'm not sure why the original isn't `eventview.query` 🤔